signals/content/subject_contains_recipient_local_part.yml

```diff
@@ -0,0 +1,4 @@
+name: "Subject: Contains Recipient Local Part"
+type: "query"
+source: |
+  any(recipients.to, strings.icontains(subject.subject, .email.local_part))
```

Comment:
Without negating an email address pattern match this will fire every time the above rule fires no? 
what we really need is to be able to do like 
`  any(recipients.to, regex.icontains(subject.subject, '.email.local_part[^@]')) but barring that 
`  
Barring that, negating an email address pattern in the subject should prevent the duplicate firing.
`and not regex.icontains(subject.subject, '.+\@.+\..+'))`

Thoughts?

Reply:
No I'm saying that 
`strings.icontains(subject.subject, .email.local_part`

Would fire on 
`subject: Congrats b_thomas@gmail.com you've won!
recipient.to: b_thomas@gmail.com`

Reply:
Since the local part is contained within the broader email address, you'll always get a duplicate firing of email_local if the full email address in subject. So in the local rule, you'd want to negate if an email pattern is in the subject.

Reply:
Oh derp yeah that's much smarter.